etc/adminhtml/system.xml

```diff
@@ -142,5 +142,24 @@
                 </field>
             </group>
         </section>
+        <section id="tax">
+            <group id="classes">
+                <field id="dynamic_shipping_tax_class" translate="label" type="select" sortOrder="11" showInDefault="1" showInWebsite="1" showInStore="0">
+                    <label>Shipping Tax Class based on cart items.</label>
+                    <comment>Highest product tax rate from cart is used as shipping tax rate when this is active.</comment>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                </field>
+            </group>
+            <group id="calculation">
+                <field id="cross_border_trade_advanced_enabled" translate="label" type="select" sortOrder="71" showInDefault="1" showInWebsite="1" showInStore="0">
+                    <label>Enable Cross Border Trade Advanced</label>
+                    <comment>When catalog price includes tax, enable this setting to fix the price for end-customers, but allow business customers to have VAT substracted.</comment>
```

Comment:
This is quite a new feature. I am fine with adding it to `MageSetup`, but I would also like to have the OK from @frostblogNet...